Directory.Build.props

```diff
@@ -6,6 +6,26 @@
     <PackageLicenseFile>LICENSE.txt</PackageLicenseFile>
   </PropertyGroup>
 
+  <!-->Since NBitcoin.Secp256k1 does not support netstandard 2.0, we will fallback to BouncyCastle build<-->
+  <PropertyGroup Condition="'$(TargetFramework)' == 'netstandard2.0'">
+    <BouncyCastle>True</BouncyCastle>
+  </PropertyGroup>
+
+
+  <Choose>
+    <When Condition="'$(BouncyCastle)'=='true'">
+      <PropertyGroup>
+        <OtherFlags>$(OtherFlags) -d:BouncyCastle</OtherFlags>
+        <DefineConstants>$(DefineConstants);BouncyCastle</DefineConstants>
+      </PropertyGroup>
+    </When>
+  </Choose>
+
+  <ItemGroup>
+    <PackageReference Include="NBitcoin" Version="5.0.53" />
+    <PackageReference Condition="'$(BouncyCastle)'=='true'" Include="Portable.BouncyCastle" Version="1.8.6.7" PrivateAssets="all" />
```

Comment:
I'd still ask why use PrivteAssets for a nuget reference?

Reply:
I once wanted to use internal projects (e.g. Macaroons, InternalBech32Encoders) from the consumer application. I required me to add `PrivateAssets="all"`

But TBH I don't understand why this solved my problem. I should do some research.
And be sure if really need it for `BouncyCastle`

Reply:
Macaroons, InternalBech32Encoders, are projects whose sources are here in DNL repo, they are not nuget references. AFAIK we add PrivateAssets attrib to them so that their DLLs are bundled in our nuget package. But BouncyCastle is a nuget dependency and doesn't need to be bundled, i.e. it should just be listed as a dep in nuget.org website.

Reply:
Yeah you are right. I've just fixed.